.github/workflows/python-package.yml

```diff
@@ -43,12 +43,6 @@ jobs:
         python -m pip install flake8 pytest pytest-mock pytest-split pytest-cov pytestify
         python -m pip install types-setuptools
         pip install .[featurizer]
-#    - name: Lint with flake8
```

Comment:
If we remove this, does flake8 still need to be installed? (Sorry, saw this only after approving)

Reply:
`flake8` is still being run because it's part of the pre-commit config (https://github.com/berquist/LobsterPy/blob/670bf72dd8224a6d71e36bcebaa7e331a50cfd5c/.pre-commit-config.yaml#L32); I didn't touch it.

In theory, ruff should cover everything flake8 plus the plugins does, but I haven't checked this.  I can if you'd like.

Reply:
Thanks, this sounds fine then. And thank you for the explanation.